src/ZfrPusher/Client/PusherClient.php

```diff
@@ -43,6 +44,15 @@ class PusherClient extends Client
     const LATEST_API_VERSION = '1.0';
 
     /**
+     * List of Pusher clusters
+     */
+    const CLUSTERS = array(
```

Comment:
Not allowed in older PHP versions it seems (the tests are failing). Maybe we could simply release a new major version with PHP 5.6+ support only? If so you'll need to update travis as well.